src/main.cc

```diff
@@ -35,9 +35,9 @@
 
 int main(int argc, const char** argv) {
   LOGFILE << "Lc0 started.";
-  CERR << "       _";
+  CERR << "\033[1;31m       _";
```

Comment:
Maybe add `utils/esc_codes.h` with something like:
```cpp
class EscCodes {
   EscCodes() = delete; // To disallow instantiation.
   static constexpr const char * kRed = "\033[1;31m";
};
```

And then use it as `CERR << EscCodes::kRed << ...`
?

It you think it's overengineering for a single use case, feel free to leave it as it is.

Reply:
Ok, maybe it's overengineering, but maybe just defining those as constants one line above is good enough.